oracle/src/events/processAMBCollectedSignatures/estimateGas.js

```diff
@@ -24,7 +25,9 @@ async function estimateGas({
     const gasEstimate = await foreignBridge.methods.executeSignatures(message, signatures).estimateGas({
       from: address
     })
-    return gasEstimate
+    const msgGasLimit = parseAMBHeader(message).gasLimit
+
+    return gasEstimate + msgGasLimit
```

Comment:
Consider to add a comment here to clarify why `+ estimateExtraGas(len)` is not needed here